packages/common/index.ts

```diff
@@ -1021,33 +1025,6 @@ export class UpdateOrganizationCourseDetailsParams {
   courseSettings?: Array<CourseSettingsRequestBody>
 }
 
-export class ChatBotQuestionParams {
-  @IsInt()
-  interactionId?: number
-
-  @IsString()
-  questionText?: string
-
-  @IsString()
-  responseText?: string
-
-  @IsBoolean()
-  suggested?: boolean
-
-  @IsInt()
-  userScore?: number
-
-  @IsString()
-  vectorStoreId?: string
-
-  @IsArray()
-  sourceDocuments?: {
-    name: string
-    type: string
-    parts: string[]
-  }[]
-}
-
```

Comment:
Good catch. This is due to the fact that createQuestion checks for these 3 properties, if not seen it will throw 400 code. 
And that also suggests that this should likely be a good opportunity to integrate this into request params. 
![image](https://github.com/user-attachments/assets/99985a3a-14d0-4df8-8b39-b9fef0e91ce4)